packages/@aws-cdk/aws-lambda/lib/alias.ts

```diff
@@ -137,6 +138,17 @@ export class Alias extends QualifiedFunctionBase implements IAlias {
       sep: ':',
     });
 
+    if (props.onFailure || props.onSuccess || props.maximumEventAge || props.maximumRetryAttemps) {
+      new EventInvokeConfig(this, 'EventInvokeConfig', {
+        function: this.version.lambda,
+        qualifier: this.aliasName,
+        onFailure: props.onFailure,
+        onSuccess: props.onSuccess,
+        maximumEventAge: props.maximumEventAge,
+        maximumRetryAttemps: props.maximumRetryAttemps
+      });
+    }
+
```

Comment:
Should we do the same for function versions as well?

Reply:
Yes but how? I don't expect users to do `new lambda.Version(...)`, the recommended approach is `fn.addVersion('1')` So maybe with a `addEventInvokeConfig(...)`?